src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/OverrideEqualsOnOverloadingOperatorEqualsTests.Fixer.cs

```diff
@@ -17,7 +19,7 @@ protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpOverrideEqualsOnOverloadingOperatorEqualsAnalyzer();
+            throw new NotSupportedException("CA2224 is not applied to C# since it already reports CS0660");
```

Comment:
If you're asking why we throw, it's so that tests don't depend on this helper.  If you're asking about the message, @srivatsn suggested that we make the analyzer VB-only since C# already has a built-in warning (CS0660) for this condition.